evaporate.js

```diff
@@ -885,7 +886,7 @@
                 }
                 var completedAt = new Date(u.completedAt || FAR_FUTURE);
 
-                return con.partSize === u.partSize && completedAt > HOURS_AGO;
+                return con.partSize === u.partSize && completedAt > HOURS_AGO && me.name === u.awsKey && con.bucket === u.bucket;
```

Comment:
just updated my above comment - I've been using key in the context of the S3 docs, which Evaporate refers to as the file name, as opposed to the Access Key